helpers/programmes_helper.py

```diff
@@ -60,5 +61,12 @@ def get_programme_choices():
     )
 
 
+def get_guild_ids():
+    if os.getenv('GUILD_IDS') == "":
+        return None
+    else:
+        return os.environ['GUILD_IDS'].split(",")
```

Comment:
I think this method should be moved to another helper, maybe something like `debug_helper`

Reply:
I'll look at that next weekend